src/main/target/MATEKF405/target.h

```diff
@@ -77,6 +77,7 @@
 #define SPI2_MOSI_PIN   	    PB15
 
 #define USE_OSD
+#define USE_MSP_DISPLAYPORT
```

Comment:
Is this `#define` needed on other targets too?

Reply:
Yes. Since I tested on the target, I only added ``#define`` to this file. I am not sure if other targets also need me to maintain.

Reply:
````common.h```` have added it.